iset.mm

```diff
@@ -169753,6 +169807,358 @@ space is a (special kind of) module, so any theorem proved below for modules
+      wi co fveq1 eqeq1d imbi2d rexralbidv elrab bitrdi ) GHTCITUAZJETJFUBZLUBU
+      CVCMUBZUCUDUEFGUFZVDSUBZUCZKUGZUNZMUHGUIUOZUFLVJUJZSAUKZTJATVEVDJUCZKUGZU
+      NZMVJUFLVJUJZUAVBEVLJABCDESFGHIKLMNOPQRULUMVKVPSJAVFJUGZVIVOLMVJVJVQVHVNV
+      EVQVGVMKVDVFJUPUQURUSUTVA $.
+  $}
+
+  ${
+    $d a b f i k r w $.
+    $( ` mPoly ` has universal domain.  (Contributed by Jim Kingdon,
+       5-Nov-2025.) $)
+    fnmpl $p |- mPoly Fn ( _V X. _V ) $=
+      ( vi vr vw vk va vb vf cvv cv cmps co cfv clt wbr wral cbs wfn wcel mp2an
+      vex c0g wceq cn0 cmap wrex crab cress csb cmpl df-mplcoe cxp fnpsr fnovex
+      wi mp3an basfn funfvex funfni rabex ressex csbexa fnmpoi ) ABHHCAIZBIZJKZ
+      CIZDIZEILVGFIZLMNDVCOVHGILVDUALUBUNFUCVCUDKZOEVIUEZGVFPLZUFZUGKZUHUICGADB
+      EFUJCVEVMJHHUKQVCHRVDHRVEHRULATBTVCVDHHJUMUOVFHRZVLHRVMHRCTZVJGVKPHQVNVKH
+      RZUPVOVPHVFPVFPUQURSUSVLVFHHUTSVAVB $.
+  $}
+
+  ${
+    mplrcl.p $e |- P = ( I mPoly R ) $.
+    mplrcl.b $e |- B = ( Base ` P ) $.
+    $( Reverse closure for the polynomial index set.  (Contributed by Stefan
+       O'Rear, 19-Mar-2015.)  (Revised by Mario Carneiro, 30-Aug-2015.) $)
+    mplrcl $p |- ( X e. B -> I e. _V ) $=
+      ( wcel cvv cmpl reldmmpl cxp wfn wrel fnmpl fnrel ax-mp relelbasov simpld
+      ) EAHDIHCIHEABJDCKJIILZMJNOTJPQFGRS $.
+  $}
+
+  ${
+    $d f B $.  $d I a b f k $.  $d R a b f k $.  $d f S $.
+    mplval2.p $e |- P = ( I mPoly R ) $.
+    mplval2.s $e |- S = ( I mPwSer R ) $.
+    mplval2.u $e |- U = ( Base ` P ) $.
+    $( Self-referential expression for the set of multivariate polynomials.
+       (Contributed by Mario Carneiro, 7-Jan-2015.)  (Revised by Mario
+       Carneiro, 2-Oct-2015.) $)
+    mplval2g $p |- ( ( I e. V /\ R e. W ) -> P = ( S |`s U ) ) $=
+      ( vk va vb vf wcel cv cfv co cress eqid wa clt wbr wral c0g wceq cn0 cmap
+      wi wrex cbs crab mplvalcoe mplbascoe oveq2d eqtr4d ) EFOBGOUAZACKPZLPQURM
+      PZQUBUCKEUDUSNPQBUEQZUFUIMUGEUHRZUDLVAUJNCUKQZULZSRCDSRVBABCVCNKEFGUTLMHI
+      VBTZUTTZVCTUMUQDVCCSVBABCDNKEFGUTLMHIVDVEJUNUOUP $.
+
+    $d B x $.  $d U x $.
+    mplbasss.b $e |- B = ( Base ` S ) $.
+    $( The set of polynomials is a subset of the set of power series.
+       (Contributed by Mario Carneiro, 7-Jan-2015.)  (Revised by Mario
+       Carneiro, 2-Oct-2015.) $)
+    mplbasss $p |- U C_ B $=
+      ( vx vk va vb vf cv wcel cfv cvv cmpl clt wbr wral c0g wceq cn0 cmap wrex
+      wi co crab wa reldmmpl cxp wfn wrel fnmpl fnrel relelbasov eqid mplbascoe
+      ax-mp syl ssrab2 eqsstrdi sseld pm2.43i ssriv ) KEAKPZEQZVIAQVJEAVIVJELPZ
+      MPRVKNPZRUAUBLFUCVLOPRCUDRZUEUINUFFUGUJZUCMVNUHZOAUKZAVJFSQCSQULEVPUEVIEB
+      TFCUMTSSUNZUOTUPUQVQTURVBGIUSABCDEOLFSSVMMNGHJVMUTIVAVCVOOAVDVEVFVGVH $.
+  $}
+
+  ${
+    $d f I $.
+    mplelf.p $e |- P = ( I mPoly R ) $.
+    mplelf.k $e |- K = ( Base ` R ) $.
+    mplelf.b $e |- B = ( Base ` P ) $.
+    mplelf.d $e |- D = { f e. ( NN0 ^m I ) | ( `' f " NN ) e. Fin } $.
```

Comment:
Why can ``D = { f e. ( NN0 ^m I ) | ( `' f " NN ) e. Fin }`` be used here, as in set.mm, and why is no substitition as made for ~df-mplcoe is required?